src/components/NotificationStack/index.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { Notification as NotificationItem } from '@santiment-network/ui'
+import { CSSTransition, TransitionGroup } from 'react-transition-group'
+import styles from './NotificationStack.module.scss'
+
+const notifyDuration = +styles.notifyDuration
+
+class NotificationStack extends Component {
+  timerHandles = {}
+
+  state = {
+    notifications: []
+  }
+
+  static getDerivedStateFromProps ({ notification }, prevState) {
+    return {
+      notifications: notification
+        ? [...prevState.notifications, notification]
+        : []
```

Comment:
Should return `null` if the state should not be changed.

Reply:
This won't have any effect as in our case, `getDerivedStateFromProps` only invokes when we have a notification props since we don't have any action like HIDE_NOTIFICATION that can unset the value of notification state